src/components/ChatEntry.js

```diff
@@ -1,22 +1,40 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
-const ChatEntry = (props) => {
+import { useState } from 'react';
+
+const ChatEntry = ({id, sender, body, timeStamp, updateLike}) => {
+  const [changeHeart, setChangeHeart] = useState('🤍');
```

Comment:
small nitpicky note - I'd recommend avoiding state variable names that sound like functions since this can be misleading. Something like `heartButton` would be a more appropriate name here.